exonum-java-binding-core/src/main/java/com/exonum/binding/service/Node.java

```diff
@@ -56,5 +56,5 @@ void submitTransaction(Transaction transaction)
    *
    * @throws IllegalStateException if the node proxy is closed
    */
-  byte[] getPublicKey();
+  PublicKey getPublicKey();
```

Comment:
A part of this task is to improve documentation

> Also clarify which of the two public keys it returns and its relationship to Blockchain#getActiveConfiguration.

Reply:
You mean service vs consensus public key? I've added this to the Javadoc.
As I understand these keys are stored in `StoredConfiguration#validatorKeys`, should I mention that in `Node#getPublicKey` Javadoc?